pkg/TIPA.py

```
@@ -158,593 +185,527 @@ class MessageFunc():
+        }
+        
+        return f"{base_urls[site]}?{urlencode(params)}"
+
+
+    def get_full_item_name(self, search_text: str, site: str) -> str:
+        if not search_text:
+            return None
+
+        try:
+            if site not in ["market", "wiki"]:
+                raise ValueError("Invalid site. Choose 'market' or 'wiki'.")
```

Ruff (0.8.0)

Avoid specifying long messages outside the exception class

(TRY003)